src/components/Editor/SearchBar.js

```diff
@@ -151,9 +156,10 @@ const SearchBar = React.createClass({
       this.setState({ enabled: true });
     }
 
-    if (this.state.functionSearchEnabled) {
+    if (this.state.symbolSearchEnabled) {
       this.clearSearch();
-      this.setState({ functionSearchEnabled: false });
+      this.setState({
+        symbolSearchEnabled: false, selectedSymbolType: "functions" });
```

Comment:
Same story, should we always reset to functions or leave the last selected symbol type like we do with previous search modifiers?